recipes/status-code/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conans import tools
+
+required_conan_version = ">=1.45.0"
+
+class StatusCodeConan(ConanFile):
+    name = "status-code"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ned14/status-code"
+    description = "Proposed SG14 status_code for the C++ standard"
+    topics = ("header-only", "proposal")
+    settings = "os", "arch", "build_type", "compiler"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _compiler_required_cpp17(self):
+        return {
+            # Their README says gcc 5, but in testing only >=7 worked
+            "gcc": "7",
+            "clang": "3.3",
+            "Visual Studio": "14",
+            "apple-clang": "5",
+        }
+
+    def validate(self):
```

Comment:
I think you are missing one thing here too https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-is-a-toolscheck_min_cppstd-call-not-enough